NetKAN/Telemachus.netkan

```diff
@@ -9,7 +9,7 @@
         "homepage" : "http://forum.kerbalspaceprogram.com/index.php?/topic/22623-104-2015-07-04-telemachus"
     },
     "ksp_version_min" : "1.0.4",
-    "ksp_version_max" : "1.0.5",
+    "ksp_version_max" : "1.2",
```

Comment:
Done.
Regarding "1.2.2": is it CKAN's philosophy to only set metadata to actually tested versions?
I thought it would be likely this will keep working for 1.2.3, if there's such a thing, but I guess we can't know before it comes out